src/host/os_chdir.c

```diff
@@ -26,6 +24,8 @@ int do_chdir(lua_State* L, const char* path)
 	z = !chdir(path);
 #endif
 
+	(void)(L);  /* warning: unused parameter */
```

Comment:
if we don't use it... why is it passed in?? I think we should just remove this argument entirely....
bigger change, but probably better long term. I think this is just a remnant for a previous PR.